scripts/simplify.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python3
```

Comment:
Need to `chmod +x` so this file can actually be ran.

Reply:
Fixed 8064a96b61e9a202657b192401c4303bf9b11ca1